### DISCUSSION TITLE
please add the half-life 2!!!

### DISCUSSION THREAD

**Discussion by nn-firc** [Gamuxorg/bbs#255] [投递和请求游戏]
游戏名称：half-life 2
游戏名称(英文)：half-life 2
下载地址 :（如果有的话）：

**Comment by shouhuanxiaoji** [comment]
em.. 站里的cs 本来是包含hl2的，当时我估计玩得人不多，就把这个给删了

**Comment by nn-firc** [comment]
emm...

**Comment by shouhuanxiaoji** [comment]
问题不大，我尽快处理一下

**Comment by nn-firc** [comment]
I can't see it.

**Comment by nn-firc** [comment]
thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "小鸡"<notifications@github.com&gt;; 
发送时间: 2020年9月22日(星期二) 晚上8:30
收件人: "Gamuxorg/bbs"<bbs@noreply.github.com&gt;; 
抄送: "nn"<2319221757@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [Gamuxorg/bbs] please add the half-life 2!!! (#109)





 
问题不大，我尽快处理一下

**Comment by nn-firc** [comment]
hello

**Comment by hyh19962008** [comment]
we'll get to it soon.
BTW, which country are you from?

Best

________________________________

**Comment by nn-firc** [comment]
china... but I use linux I don't have chines fonts...

**Comment by nn-firc** [comment]
chinese
